examples/shapes/main.go

```diff
@@ -0,0 +1,248 @@
+// Example shapes shows how to draw basic shapes into a window.
```

Comment:
Could you move this example to new PR please? This one is called ```Added an explanation of how to use events to the create-window example``` and should contain only changes related to this PR.

Reply:
#3

Reply:
257276c